src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -304,6 +304,12 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             var functionDetails = await WebFunctionsManager.GetFunctionMetadataResponse(listableFunctions, hostOptions, _hostNameProvider);
             result.Add("functions", new JArray(functionDetails.Select(p => JObject.FromObject(p))));
 
+            // Add host.json to the payload
+            if (_environment.IsKubernetesManagedHosting())
+            {
+                result.Add("host.json", GetHostJson());
```

Comment:
See the comment I left in the previous iteration. Looks like a revision you posted blew the comment away.

Reply:
Let's not make this Kubernetes specific, since part of our larger goal is to address https://github.com/Azure/azure-functions-host/issues/7288. Instead, I think we should add just the host.json "extensions" section here, and below if we revert to minimal format based on size, we redefine the minimal format as `{ triggers: [], extensions: {} }` ensuring this data is present in that case as well.

Reply:
Thank you @mathewc  
Done. 
Could you have a look?

Reply:
Looks good. I just had one more comment that I added on removing the unnecessary log statement. They we can get this checked in after you add a test for the scenario - I believe you're in the process of adding one?

Reply:
Thank you @mathewc 
I tested E2E, so that code is good for now, I'm planning add unit testing for it. 
Do we need E2E testing as well? If so I'd happy to do that.

Reply:
For the scope of the changes here unit test is enough! Thanks!

Reply:
I added an Unit Test. Looks good?

Reply:
I don’t see updates to unit tests. Missed a commit?

Reply:
Hi @pragnagopa 
Sorry, I missed. Now added.